server/graphql/v2/mutation/UserMutations.js

```diff
@@ -0,0 +1,30 @@
+import { GraphQLNonNull } from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import { Unauthorized, ValidationFailed } from '../../errors';
+import { fetchAccountWithReference } from '../input/AccountReferenceInput';
+import { Individual } from '../object/Individual';
+
+const userMutations = {
+  setChangelogViewDate: {
+    type: new GraphQLNonNull(Individual),
+    description: 'Update the time which the user viewed the changelog updates',
+    args: {
+      changelogViewDate: {
+        type: new GraphQLNonNull(GraphQLDateTime),
+      },
+    },
+    resolve: async (_, { changelogViewDate }, { remoteUser }) => {
+      if (!remoteUser) {
+        throw new Unauthorized();
+      } else if (!changelogViewDate) {
+        throw new ValidationFailed('The change log view date must be set');
+      }
+      const date = new Date(changelogViewDate);
+      const user = await remoteUser.update({ changelogViewDate: date });
+      return await fetchAccountWithReference({ id: user.CollectiveId });
```

Comment:
Not needed

```suggestion
      return fetchAccountWithReference({ id: user.CollectiveId });
```

Reply:
Works but using fetchAccountWithReference is not proper, we can simply do Collective.findByPk.

`fetchAccountWithReference` main use case is when we pass it a `AccountReferenceInput`

Reply:
Or even better `user.getCollective()`

Reply:
Thanks for the feedback. Done. 👍🏽